src/routes/contents/[id]/+page.svelte

```diff
@@ -76,11 +99,20 @@
     }
 
     function setReference(item) {
+        let docSet;
         let book;
         let chapter;
         let verse;
         const reference = item.linkTarget.split('.');
 
+        if (item.layoutMode && item.layoutCollection?.length > 0) {
```

Comment:
Add a comment about treating all layoutModes at single since that is all we support at the moment.